docs/user/troubleshooting.md

```diff
@@ -53,20 +53,20 @@ sudo crictl pods --name=etcd-member
 
 If no pods are shown, etcd will need to be [investigated](#etcd-is-not-running).
 
-### Unable to SSH into Master Nodes
+### Unable to SSH into Control Plane Nodes
```

Comment:
Since #1045, there's no significant difference between control-plane and compute machines.  Maybe this should just be `### Unable to SSH into Nodes`?